Staging (#138)
makerdao/endgame-docs: maker-core/list-of-scopes.md

# List of Scopes

{% hint style="warning" %} This documentation describes ongoing changes in functionality and processes. Be aware that parts may be inaccurate or out of date. {% endhint %}

> Here is a list of the five Scopes:

Reviewer: ```suggestion
The Maker Protocol has five Scopes. They are:
```

Reviewer 2: We're being inconsistent with the capitalization of "scopes".

"Scopes" seems better
